trillian/docs/ManualDeployment.md

```diff
@@ -0,0 +1,460 @@
+    43:d0:b5:97:72
+ASN1 OID: prime256v1
+NIST CURVE: P-256
+```
+
+**Cross-check**: Once the CTFE is configured and running, the `ctclient`
+command-line tool allows signature checking against the public key with
+the `--pub_key` option:
+```bash
+% go install github.com/google/certificate-transparency-go/client/ctclient
+% ctclient --log_uri http://localhost:6966/aramis --pub_key pubkey.pem sth
+2018-10-12 11:28:08.544 +0100 BST (timestamp 1539340088544): Got STH for V1 log (size=11718) at http://localhost:6966/aramis, hash 6fb36fcca60d61aa85e04ff0c34a87782f12d08568118602eec0208d85c3a40d
+Signature: Hash=SHA256 Sign=ECDSA
+Value=3045022100df855f0fd097a45070e2eb244c7cb63effda942f2d30308e3b84a72e1d16118b0220038e55f142501402cf03790b3997081f82ffe47f2d3f3b667e1c484aecf40a33
+```
+
+### CA Certificates
+
+Each Log must decide on its own policy about which CA's certificates are to be
+included in the Log; this section therefore just provides an *example* of the
+process of configuring this set for the CT Log software.
+
+On a Debian-based system, the `ca-certificates` package includes a collection
+of CA certificates under `/etc/ssl/certs/`.  A set of certificates suitable
+for feeding to `ct-server` can thus be produced with:
+
+```bash
+% sudo apt-get install -qy ca-certificates
+% sudo update-ca-certificates
+% cat /etc/ssl/certs/* > ca-roots.pem
+```
+
+**Cross-check**: Once the CTFE is configured and running, opening
+`http://localhost:<port>/<prefix>/ct/v1/get-roots` shows the configured roots.
+Alternatively, the `ctclient` command-line tool shows the same information
+in a more friendly way:
+```bash
+% go install github.com/google/certificate-transparency-go/client/ctclient
+% ctclient --log_uri http://localhost:6966/aramis getroots
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 67554046 (0x406cafe)
+    Signature Algorithm: ECDSA-SHA256
+...
+```
+
+
+### CTFE Configuration
+
+The information from the previous steps now needs to be assembled into a
+configuration file for the CTFE, in
+[text protocol buffer format](https://developers.google.com/protocol-buffers/docs/overview).
+
+Each Log instance needs configuration for:
+ - `log_id`: The Trillian tree ID from an [earlier step](#tree-provisioning)
+ - `prefix`: The path prefix the log will be served at.
+ - `max_merge_delay_sec`: The MMD for the log (typically 86400, which is 24 hours).
+ - `roots_pem_file`: The files holding
+   [accepted root CA certificates](#ca-certificates) (repeated).
+ - `private_key`: The [private key](#key-generation) for the log instance.
+    For a private key held in an external PEM file, this is of the form:
+     ```
+     private_key: {
+       [type.googleapis.com/keyspb.PEMKeyFile] {
+         path: "privkey.pem"
+       }
+     }
+     ```
+ - `public_key`: The corresponding public key for the log instance.  (This is
+   not actually used by the CTFE, but is worth including for reference and for
+   use by test tools.)
```

Comment:
Worth a one-liner that ctfe supports multi tenancy too?

Reply:
Added higher up.